cimr/processor/utils.py

```diff
@@ -12,6 +12,25 @@
 import logging
 import subprocess
 
+def set_chrom_dict():
+    """Make a dictionary to standardize chromosome IDs in input files."""
+    maxchrom = 23
+    chrom_dict = {str(i):'chr' + str(i) for i in range(1, maxchrom)}
+    chrom_dict.update({'chr'+str(i):'chr' + str(i) for i in range(1, maxchrom)})
+    chrom_dict.update({
+        'X':'chr23', 
+        'Y':'chr24', 
+        'M':'chr25', 
+        'MT':'chr25', 
+        'chrX':'chr23', 
+        'chrY':'chr24', 
+        'chrM':'chr25', 
+        'chrMT':'chr25'
+    })
+    chrom_str = ['chr' + str(i) for i in range(1, maxchrom)]
+    chrom_int = [i for i in range(1, maxchrom)]
```

Comment:
can't you just call: list(range(1, maxchrom))?

Reply:
you are right